site/source/tutorials/1030_routing/index.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: tutorial
+title: Routing
+overview: In this tutorial, you will learn how to use Dojo 2's declarative routing within your application.
+---
+
+{% section 'first' %}
+
+# Application routing
+
+## Overview
+
+Dojo 2's routing is a powerful set of tools to support declarative routing using a high order component, an `Outlet` and a component that creates links with a `href` generated from an `outlet` name.
+
+In this tutorial, we will start with a basic application with no routing. We will use Dojo 2's declarative routing to configure some routes, create `outlets` from our existing widgets and use the `Link` component to create links for the application outlets.
+
+## Prerequisites
+You can [download](../assets/1030_routing-initial.zip) the demo project and run `npm install` to get started.
+
+The `@dojo/cli` command line tool should be installed globally. Refer to the [Dojo 2 local installation](../000_local_installation/) article for more information.
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
+
+{% section %}
+
+## Configuring the router
+
+{% task 'Create the routing configuration' %}
+
+All application routes needs to be to configured when creating the router instance otherwise, entering a route will not trigger a transition within the application. The `RouteConfig` a basic object consisting off `path`, `outlet`, `defaultParams`, `defaultRoute` and `children`.
+
+The routing hierarchy is generated using by adding an array `RouteConfigs` to the children property, which get recursively processed, building up the application routing paths.
+
+{% instruction 'Include the required imports `main.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' lines:2,3 %}
+
+{% instruction 'Define the routing configuration`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' lines:9-31 %}
+
+{% instruction 'Now, register the router in a `registry`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' lines:33-34 %}
+
+A helper utility above, is provided by `@dojo/routing`, that can be used to create a routing instance. The util accepts the applications routing configuration, a `registry` to define a `router` injector against and returns the `router` instance.
```

Comment:
The util accepts -> The utility accepts

Reply:
", a `registry` to define a `router` injector against and returns the `router` instance." -> ", and a `registry` to define a `router` injector against. The utility returns the `router` instance."